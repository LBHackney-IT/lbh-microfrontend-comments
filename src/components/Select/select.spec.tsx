import React from 'react';
import { render } from '@testing-library/react';

import { Select } from '@components';

test('it renders correctly', () => {
    const { container } = render(<Select />);
    expect(container).toMatchSnapshot();
});

test('it renders correctly with error', () => {
    const { container } = render(<Select error />);
    expect(container).toMatchSnapshot();
});

test('it renders correctly with isFullWidth', () => {
    const { container } = render(<Select isFullWidth />);
    expect(container).toMatchSnapshot();
});
