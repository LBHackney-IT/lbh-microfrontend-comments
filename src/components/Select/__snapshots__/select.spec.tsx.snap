// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div>
  <select
    class="govuk-select lbh-select"
  />
</div>
`;

exports[`it renders correctly with error 1`] = `
<div>
  <select
    class="govuk-select lbh-select govuk-select--error"
  />
</div>
`;

exports[`it renders correctly with isFullWidth 1`] = `
<div>
  <select
    class="govuk-select govuk-!-width-full lbh-select"
  />
</div>
`;
